parts/xmem_nozzle.go

```diff
@@ -869,7 +890,16 @@ func (xmem *XmemNozzle) Start(settings metadata.ReplicationSettingsMap) error {
 		return err
 	}
 
-	xmem.Logger().Infof("%v has been started", xmem.Id())
+	if xmem.source_cr_mode == base.CRMode_Custom || xmem.config.crossClusterVers {
```

Comment:
All these needs to be done before `xmem.start_time` is set as part of the initialization accounting

Reply:
Done